standalone-tools/wss-scan.config

```diff
@@ -0,0 +1,3 @@
+baseDirPath=$(pwd)
+gitBasePath=https://github.com/opendistro-for-elasticsearch/
+gitRepos=alerting,simple-ingest-transformation-utility-pipeline,opendistro-build,security,performance-analyzer-rca,anomaly-detection,index-management,security-kibana-plugin,sql,job-scheduler,kibana-reports,notifications,performance-analyzer,index-management-kibana-plugin,alerting-kibana-plugin,common-utils,k-NN,anomaly-detection-kibana-plugin,perftop,kibana-notebooks,sql-workbench,sql-cli,sql-jdbc,sql-odbc,piped-processing-language
```

Comment:
sql-workbench,sql-cli,sql-jdbc,sql-odbc have been merged into the sql repository so they can be removed from this list